client/src/features/dashboardV2/DashboardV2.module.scss

```
@@ -0,0 +1,43 @@
+@import "~bootstrap/scss/functions";
+@import "~bootstrap/scss/variables";
+@import "~bootstrap/scss/variables-dark";
+@import "~bootstrap/scss/maps";
+@import "~bootstrap/scss/mixins";
+
+.DashboardHeader {
+  height: auto;
+
+  img {
+    height: 150px;
+
+    @include media-breakpoint-up(md) {
+      height: 180px;
+    }
+
+    @include media-breakpoint-up(lg) {
+      height: 130px;
+    }
+  }
+}
+
+.AnonymousAvatar {
+  width: 64px;
+  height: 64px;
+}
+
+.DashboardCard {
+  background-color: rgba(var(--bs-primary-rgb), 0.1) !important;
+
+  &:hover {
+    background-color: var(--bs-primary-bg-subtle) !important;
+  }
+}
+
+.FooterCard {
+  a svg {
+    transition: all 0.3s linear;
+  }
+  a:hover svg {
```

Small detail, but this would be nice:

```suggestion
  a:hover svg, a:focus svg {
```

Reference: https://webaim.org/techniques/hypertext/link_text#hover